raw_socket_listener/tcp_message.go

```diff
@@ -92,9 +92,14 @@ func (t *TCPMessage) IsMultipart() bool {
 	}
 
 	payload := t.packets[0].Data
-	m := payload[:4]
```

Comment:
Thanks! But i prefer to have guard statement here:
```
if len(payload) < 4 {
  return false
}
```

Will be great if you can update it like this.